### DISCUSSION TITLE
branding

### DISCUSSION THREAD

**Discussion by everythingness** [flowershow/flowershow#602] [General and Help] [answered]
hi is it possible to get rid of the 'created by' and the branding at the bottom of the page? i'd like to tell my reader that it's made with flowershow on the about page not to have it in their face on every page on the site...

**Comment by rufuspollock** [comment, marked as answer]
@everythingness i think there is a feature request for this and for now i think you could edit the main layout template at https://github.com/datopian/flowershow/blob/main/components/Layout.tsx#L149C21-L149C65 (you'd just have to insert your own footer html yourself)

**Reply by everythingness** [reply]
thanks!

**Comment by everythingness** [comment]
woohoo! beautiful. thank you! i have NFI what i'm doing but i just 
removed all mentions of 'footer' from the 'layout.tsx' in the deployment 
and it works! :)

**Reply by rufuspollock** [reply]
Great!
